spec/services/solidus_bolt/accounts/detail_service_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::Accounts::DetailService, :vcr, :bolt_configuration do
+  describe '#call', vcr: true do
+    context 'with wrong access_token' do
+      subject(:api) { described_class.new(access_token: access_token) }
```

Comment:
this and line 18 should be extracted to between line 6 and 7

Reply:
fixed here
[4816aeb](https://github.com/nebulab/solidus_bolt/commit/4816aeb0685d59f8bd9e005454b69ef638010228)